modules/ROOT/pages/design-common-problems-raml-10.adoc

```diff
@@ -284,10 +288,56 @@ Name:
       a: "b"
       b: 1
 ----
+////
+
+=== Naming examples twice
```

Comment:
@kbostrom , it's complicated. For two releases of API Designer, the UI has been able to display the parent section when a) a user receives one of the error messages that the section contains, and b) the user clicks an info icon at the end of the message. Originally, there were several error messages that the parent section included. However, because of improvements in the parser that the developers use to find errors in API specifications, two of the error messages no longer appear. Those are the error messages that I commented out. The parser does, however, now find the message that I added. This is all to say that I cannot bump up the heading for this new error message because it would not be displayed in the UI when a user clicks that info icon. The heading for the parent section contains some special anchor that the dev team uses to locate the section and pull it into the UI. Sorry for the longwinded explanation, but I don't have time to make it more concise.